awx/main/dispatch/worker/callback.py

```diff
@@ -104,7 +133,14 @@ def _save_event_data():
             retries = 0
             while retries <= self.MAX_RETRIES:
                 try:
-                    _save_event_data()
+                    kwargs = cls.create_from_data(**body)
+                    workflow_job_id = kwargs.pop('workflow_job_id', None)
```

Comment:
`workflow_job_id` isn't an actual column, so we can't pass it to the event `__init__()`.

Instead, we use `setattr` on the instance so that message sent to external loggers contains it.